implement-shell-tools/ls/ls.py

```diff
@@ -0,0 +1,47 @@
+import os
+import argparse
+import locale
+
+
+#set locale to the machine default setting
+locale.setlocale(locale.LC_ALL, '')
```

Comment:
I'm not familiar with this function.  It seems odd that you're having to set something to its default.  If it's the default, isn't it already set?